.mvn/settings.xml

```diff
@@ -6,7 +6,8 @@
     <mirror>
       <id>BetonQuest-Mirror</id>
       <url>https://nexus.betonquest.org/repository/default/</url>
-      <mirrorOf>*</mirrorOf>
+      <!--      <mirrorOf>*</mirrorOf>-->
+      <mirrorOf>*,!betonquest-nexomc-repo</mirrorOf>
```

Comment:
Tet documentation only explains this for local testing, this should also not be checked in